Test Cases/Generate Code/Data-Driven Testing/6. Generate Data-Driven Script.tc

```diff
@@ -30,7 +30,7 @@
       <variableId>205813e2-03fe-4427-804d-012ddd18aa23</variableId>
    </variableLinks>
    <variable>
-      <defaultValue>''</defaultValue>
+      <defaultValue>'Tokyo CURA Healthcare Center'</defaultValue>
```

Comment:
It should be `''` because later when executing the test case, we will fill its values with data from csv file